src/mixins/nft.js

```diff
@@ -647,6 +648,26 @@ export default {
         );
       }
 
+      if (this.nftIsNFTBook) {
+        const sortedList = [...recommendedList];
+        sortedList.sort((a, b) => {
+          if (
+            BOOK_NFT_SYMBOL.includes(a.symbol) &&
+            !BOOK_NFT_SYMBOL.includes(b.symbol)
```

Comment:
Should judge if nft is book from class metadata first, instead of relying only on symbol

Reply:
[ea433ae](https://github.com/likecoin/liker-land/pull/1490/commits/ea433ae055b1374ab75410e2793888a82c6a7c98)